### DISCUSSION TITLE
How do I change monitor's ICC color profile?

### DISCUSSION THREAD

**Discussion by LonelyMusicDisc** [i3/i3#6161] [Q&A] [answered]
I don't think gnome-color-manager would work since in the website it only mentions the gnome environment.

**Comment by KebabLord** [comment, marked as answer]
i'm using dispcalgui (displaycal) which works on i3 and has ICC profile management features.
![image](https://github.com/user-attachments/assets/30802c3d-90f2-43f3-871d-0cb720d88fec)
